src/standard/x-styling.html

```diff
@@ -129,7 +131,7 @@
         // finally mixin properties inherent to this element
         this.mixin(props,
           propertyUtils.scopePropertiesFromStyles(this._styles));
-        this.mixin(props, this.customStyle);
+        propertyUtils.mixinCustomStyle(props, this.customStyle);

```

Comment:
Actually, it seems like `mixinCustomStyle`'s behavior on `null` should not delete properties that came from the scope... i.e. if you set `this.customStyle['--foo'] = 'red';`, that should override `:root`, but `this.customStyle['--foo'] = null` should just let any props in `:root` through without overriding with `null`.